portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,14 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const randomFacts =
+      ['I like to code.', 'Soccer is my favorite sport.', 'My favorite color is blue.', 'I live in Mexico.'];
```

Comment:
nit: Please break the line to fit in 100 characters.
Each item in the array can be on a separate line.

Reply:
Ok, will do!